R/api.R

```diff
@@ -366,6 +367,18 @@ endpoint_adr_metadata <- function(queue) {
                                     returning = response)
 }
 
+
+endpoint_upload_input <- function(queue) {
+  input <- porcelain::porcelain_input_body_binary("file")
+  response <- porcelain::porcelain_returning_json(
+    "SessionFile.schema", schema_root())
+  porcelain::porcelain_endpoint$new("POST",
+                                    "/upload/input/<filename>",
```

Comment:
I wonder about putting this below some other top level path (internal/upload perhaps) if there are many to make it clear it's not really part of the public API?

Reply:
Yeah sounds like a good idea